pallets/accounts/src/lib.rs

```diff
@@ -758,6 +818,15 @@ pub mod pallet {
 			}
 		}
 
+		fn apply_20_percent(amount: u128) -> u128 {
```

Comment:
20% should not be hardcoded.

Reply:
This function is temporary, I need to implement a better way to retrieve all the vesting rules with another PR (issue #174 ).
The float (in another comment above) is needed here indeed.